evap/contributor/templates/contributor_index.html

```diff
@@ -157,12 +157,26 @@
                                                         <span class="fas fa-pencil"></span>
                                                     </a>
                                                     {% if not evaluation|has_nonresponsible_editor %}
-                                                        <a href="#" class="btn btn-sm btn-dark" data-bs-toggle="tooltip"
-                                                            data-bs-placement="top" title="{% trans 'Delegate preparation' %}"
-                                                            onclick="delegateSelectionModalShow(`{{ evaluation.full_name }}`, `{% url 'contributor:evaluation_direct_delegation' evaluation.id %}`);return false;"
-                                                        >
-                                                            <span class="fas fa-hand-point-left"></span>
-                                                        </a>
+                                                        <form method="POST" action="{% url 'contributor:evaluation_direct_delegation' evaluation.id %}" class="d-inline">
+                                                            {% csrf_token %}
+
+                                                            <confirmation-modal type="submit">
+                                                                <span slot="title">{% trans 'Delegate preparation' %}</span>
+                                                                <span slot="action-text">{% trans 'Delegate preparation' %}</span>
+                                                                <span slot="question">
+                                                                    {% blocktrans with evaluation_name=evaluation.full_name %}
+                                                                        Do you really want to delegate the preparation of the evaluation {{ evaluation_name }}?
+                                                                    {% endblocktrans %}
+                                                                    <div class="my-4">
+                                                                        {% include 'bootstrap_form.html' with form=delegate_selection_form wide=True %}
```

Comment:
The dropdown is only shown directly when using the arrow or dragging the input out of the modal.
Also, this modal closed, when the submit button is clicked without any selection. The tooltip is shown for a short time after the modal is closed